avocado/plugins/config.py

```diff
@@ -100,8 +100,8 @@ def handle_default():
             LOG_UI.debug("file to customize values")
             LOG_UI.debug('')
             LOG_UI.info('Avocado Data Directories:')
-            LOG_UI.debug('    base     %s', data_dir.get_base_dir())
-            LOG_UI.debug('    tests    %s', data_dir.get_test_dir())
-            LOG_UI.debug('    data     %s', data_dir.get_data_dir())
-            LOG_UI.debug('    logs     %s', data_dir.get_logs_dir())
-            LOG_UI.debug('    cache    %s', ", ".join(cache_dirs))
```

Comment:
Out of curiosity: why did you remove one of the leading spaces here?

Reply:
Just to keep less then 80 columns.